include/nbl/builtin/glsl/ext/CentralLimitBoxBlur/parameters.glsl

```diff
@@ -0,0 +1,64 @@
+#ifndef _NBL_GLSL_EXT_BLUR_PARAMETERS_INCLUDED_
+#define _NBL_GLSL_EXT_BLUR_PARAMETERS_INCLUDED_
+
+#include "nbl/builtin/glsl/ext/CentralLimitBoxBlur/parameters_struct.glsl"
+
+#ifndef _NBL_GLSL_EXT_BLUR_GET_PARAMETERS_DECLARED_
+nbl_glsl_ext_Blur_Parameters_t nbl_glsl_ext_Blur_getParameters();
+#define _NBL_GLSL_EXT_BLUR_GET_PARAMETERS_DECLARED_
+#endif
+
+#ifndef _NBL_GLSL_EXT_BLUR_PARAMETERS_METHODS_DEFINED_
```

Comment:
they should have a forward declaration in all honesty

Reply:
I don't get it, forward declared where?

Reply:
either you forward declare the getters, like this
```cpp
uvec3 nbl_glsl_ext_Blur_Parameters_t_getDimensions();
```
and define later under an ifdef like you currently do

or do like we currently do for "pseudo methods" (stuff that could be struct methods in HLSL or C)
```
uvec3 nbl_glsl_ext_Blur_Parameters_t_getDimensions(in nbl_glsl_ext_Blur_Parameters_t this)
{
    return this.input_dimensions.xyz;
}
```